app/models/zipped_moab_version.rb

```diff
@@ -35,7 +35,7 @@ def all_parts_replicated?
     # the assumption is that all of the database part records are created at once,
     # initialized to 'unreplicated', as soon as the (possibly multi-part) zip file
     # has been created and completely written to disk.  see DruidVersionZip.
-    zip_parts.count.positive? && zip_parts.all?(&:ok?)
+    zip_parts.any? && zip_parts.all?(&:ok?)
```

Comment:
Any reason to be skeptical of these `#count`-related changes? They are motivated by Rubocop, so we can roll them back if we're concerned about changes in behavior.

Reply:
Yes. This might result in a less efficient SQL query.